save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/vuln/VulnerabilityView.kt

```diff
@@ -172,3 +177,8 @@ private fun ChildrenBuilder.languageSpan(language: VulnerabilityLanguage) {
         +language.value
     }
 }
+
+internal fun hasRightsToEdit(currentUserInfo: UserInfo?, vulnerability: VulnerabilityDto) =
+    // globalRole is enriched on the App.kt level with "users/global-role" (based on ROLE from spring security authority)
+    // and is not stored in DB, so we cannot check userInfo directly, because in the getAllParticipants() users will have this field null
```

Comment:
globalRole is `User.role`

Reply:
Yes and no. GlobalRole is not simple set from role - basically enriched from authority. But initial value stored in User.role, yes. 

But in this comment I am pointing that globalRole is not set in UserDetails directly and details are only enriched later with this value on the frontend with a separate request

Reply:
raised to look at it later: https://github.com/saveourtool/save-cloud/issues/2540